### PR TITLE
2016-03-18 Weekly

### PR DESCRIPTION
**Pull request by kimtryce**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-b0b0dd47a1/)

File: _posts/2016-03-18 Weekly-News.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Weekly news: Backups, patching security vulnerabilities, scheduling tasks and a testimonial"
+date: 2016-03-18 13:00
+comments: true
+author: Kim Tryce <kim.tryce@rackspace.com>
+published: true
+excerpt: >
+Some important updates including how to back up and restore container data, patching security vulnerabilities with Watchtower, and scheduling tasks with a cron container.  We also have a guest post by a Rackspace team who use Carina to run their ChatOps operation.
```
Comment (ktbartholomew) [inline _posts/2016-03-18 Weekly-News.md]: I would make this a complete sentence.

> This week we have some important updates, including...

File: _posts/2016-03-18 Weekly-News.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+authorIsRacker: true
+---
+
+Follow the rainbow to the pot of gold and check out the St. Patrick's Day edition of Carina: Operational Maturity!  We have a few tutorials that lead you down the path towards a backedup and secure system.  To wrap it up, one of our product teams at the Rack decided to use Carina to run a high dependency tool, Chatops.
+
+### Tutorial: Patching security vulnerabilities with Watchtower
+
+Who wants to get hacked?! Security + Vulnerabilities = Buzzwords, but important ones.  Yes, it's true folks. Security vulnerabilities exist and it's important to keep your systems up to date with the latest versions.  This protects your investment of time/energy/money.  [Jamie](https://twitter.com/jamiehannaford) provides an [introduction tutorial](https://getcarina.com/docs/tutorials/patching-security-vulnerabilities/) on using [Watchtower](https://github.com/getcarina/watchtower) to detect, patch, and redeploy containers running out-of-date [Docker](https://www.docker.com/) images.
+
+![Follow the rainbow]({% asset_path 2016-03-18-Weekly-News/security-rainbow.jpg %})
```
Comment (ktbartholomew) [inline _posts/2016-03-18 Weekly-News.md]: Do the files for these images exist somewhere? They're not in the PR.

File: _posts/2016-03-18 Weekly-News.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+![Backups!]({% asset_path 2016-03-18-Weekly-News/backup-all-the-things.jpg %})
+
+### Tutorial: Schedule tasks with a cron container
+
+To follow up [Keith's tutorial](https://getcarina.com/docs/tutorials/backup-restore-data/) on backing up your data, he goes the extra mile to help with the next step: automation! In [this tutorial](https://getcarina.com/docs/tutorials/schedule-tasks-cron/) Keith teaches you how to run cron job using containers to wrap up the operational aspects of protecting your backups.  But chron jobs are not just for backups! For the devops folks out there running systems and maintain environments, this tutorial is for you.    
+
+### User Story: Moving ChatOps Into Carina
+
+Our own [Nick](https://twitter.com/filler) from the Cloud DNS product team weaves [the tale](https://getcarina.com/blog/moving-chatops-into-carina/) of how the DNS ops team picked up their entire ChatOps operation running a Hubot instance on Heroku with a RedisToGo redis-based datastore and move the Hubot instance to Carina using Docker Compose and OjectRocket redis datastore. And it took one hour. He also touches on the `makefile` command and its usefullness.
```
Comment (ktbartholomew) [inline _posts/2016-03-18 Weekly-News.md]: This sentence seems to wander? Maybe this:

> ...weaves the tale of how Rackspace's DNS team moved their entire ChatOps platform from Heroku to Carina in _one hour_.

**Comment by ktbartholomew** [comment]
@kimtryce this post isn't appearing in the PR builder preview, I suspect because the date/time of the post is in the future (13:00 today). I think if you change the post's date to be _only_ the date, it should appear so we can make sure everything looks right before shipping.

File: _posts/2016-03-18 Weekly-News.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+One of the most important aspects of your data is backing it up!  As Carina is in beta, we want to make sure that our users don't have a bad day and find out there has been a data loss.  In order to prevent this from happening, [Keith](https://twitter.com/ktbartholomew) provides an [example](https://getcarina.com/docs/tutorials/backup-restore-data/) using MySQL and provides concepts that can be utlized for any backup.  Knowing you have your data backed up = a good night's sleep.
+
+![Backups!]({% asset_path 2016-03-18-Weekly-News/backup-all-the-things.jpg %})
+
+### Tutorial: Schedule tasks with a cron container
+
+To follow up [Keith's tutorial](https://getcarina.com/docs/tutorials/backup-restore-data/) on backing up your data, he goes the extra mile to help with the next step: automation! In [this tutorial](https://getcarina.com/docs/tutorials/schedule-tasks-cron/) Keith teaches you how to run cron job using containers to wrap up the operational aspects of protecting your backups.  But chron jobs are not just for backups! For the devops folks out there running systems and maintain environments, this tutorial is for you.    
+
+### User Story: Moving ChatOps Into Carina
```
Comment (everett-toews) [inline _posts/2016-03-18 Weekly-News.md]: Users are most important to us. This should be at the top IMO.

**Comment by kmsholcomb** [comment]
Hi @kimtryce: The attached Word doc (and PDF copy of it) contain my suggested edits for the content. I also tried to incorporate/address most of Keith's and Everett's feedback as well. Please let me know if you have any questions. Thanks!

[weekly new 3-18-2016_editsKH.pdf](https://github.com/getcarina/getcarina.com/files/180096/weekly.new.3-18-2016_editsKH.pdf)
[weekly new 3-18-2016_editsKH.docx](https://github.com/getcarina/getcarina.com/files/180097/weekly.new.3-18-2016_editsKH.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-cab69bf5cd/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-f8e1095818/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-c1ea6fe88d/)

**Comment by ktbartholomew** [comment]
@kimtryce The post still isn't being build in the PR builder. Now I suspect the file name (there's a space in it) may be the cause. Please rename the file to `2016-03-18-weekly-news-chatops.md`.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-cc737833eb/)

File: _posts/2016-03-18-weekly-news-chatops.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+ - Cron
+ - News
+authorIsRacker: true
+---
+
+Follow the rainbow to the pot of gold and check out the St. Patrick's Day edition of Carina: Operational Maturity!  To start it off, one of the Rackspace product teams decided to use Carina to run a high dependency tool, ChatOps.   We also have a few tutorials that lead you down the path toward a backed up and secure system.
+
+### User story: Moving ChatOps into Carina
+
+Our own [Nick](https://twitter.com/filler) from the Cloud DNS product team tells [the tale](https://getcarina.com/blog/moving-chatops-into-carina/) of how the team moved their entire ChatOps platform from Heroku to Carina in <i>one hour</i>. Nick also talks about the `makefile` command and its usefulness.
```
Comment (ktbartholomew) [inline _posts/2016-03-18-weekly-news-chatops.md]: This isn't worth changing, but you can express the italics in this sentence with underscores if you wish: `in _one hour_.`

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-ce64b8b8cd/)

**Comment by ktbartholomew** [comment]
LGTM :+1: :shipit: thundercats are go

**Pull request merged**